docs/content/docs/mokksy/file-config.md

```
@@ -0,0 +1,221 @@
+    method: GET
+    path: /ping
+    response:
+      body: '{"response":"Pong"}'
+      status: 200
+```
+
+Load the file and start the server:
+
+{{< code-tabs >}}
+{{< tab lang="kotlin" >}}
+```kotlin
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)